git_cheatsheet.md

```diff
@@ -0,0 +1,104 @@
+# Git Cheatsheet
```

Comment:
Should we tell them that these commands are mostly used in the GitBash app?

Reply:
@GBruening they work with powershell/terminal as well.